datafusion/physical-expr/src/expressions/case/mod.rs

```diff
@@ -15,9 +15,9 @@
 // specific language governing permissions and limitations
 // under the License.
 
-use super::{Column, Literal};
-use crate::expressions::case::ResultState::{Complete, Empty, Partial};
-use crate::expressions::try_cast;
+mod literal_lookup_table;
```

Comment:
I asked a bit and the preference seems to be to avoid mod.rs files with lots of code in it. Could we move this file back to case.rs?